### DISCUSSION TITLE
Foundry V10 ETA

### DISCUSSION THREAD

**Discussion by jesuisfrog** [3darkman/foundryvtt-chroniclesystem#27] [General]
First of all, thank you for creating a foundry version of the system! 

I was wondering if you had an idea as to when the system would be usable in V10.

**Comment by 3darkman** [comment]
From what I've tested of it, it's working with v10. But there are some
warnings that it will stop working in v12, so I'm already making these
corrections.

Em ter., 20 de set. de 2022 13:27, Michaël B. ***@***.***>
escreveu:

**Reply by jesuisfrog** [reply]
I had tested the system 2 weeks ago and the sheets were broken, all the sections where you could "add" (+) elements in were blank without the option to add anything.

Was that a v10 issue that was then resolved ? 

I just checked and here's what I get. 

![image](https://user-images.githubusercontent.com/7178778/191317696-57a562f8-ff7f-4b9f-87ee-d88498619ded.png)
![image](https://user-images.githubusercontent.com/7178778/191317844-5a300606-6c91-43f0-8a1a-72708ed4bcda.png)
![image](https://user-images.githubusercontent.com/7178778/191317868-026a8684-1f11-4596-a208-ea0b0668552f.png)
![image](https://user-images.githubusercontent.com/7178778/191317905-228ee0f6-97c6-457f-b6cb-c5dbd8f869b6.png)

**Reply by avorial** [reply]
You need to build the abilities/weapons/ect as items and then drag them on the character you want to put them in. They are not hard coded as default to allow ease of customization of  abilities. https://i.imgur.com/2rGxoba.png
https://i.imgur.com/HcO6Yc0.png

**Reply by avorial** [reply]
These are the "items" I made, so if you want to make a new character you just drag the entire "abilities" folder on the character and now they have those abilities. 

Then you drag each of their items, and then any qualities or drawbacks they have. 

The same works for the house sheet for Holdings.

![chrome_uSg0D6y4OU](https://user-images.githubusercontent.com/65983414/191319820-33560110-3f0f-4ed3-8fef-11cc94a62815.png)

**Reply by jesuisfrog** [reply]
Oh ! I see, I was not expecting that, might be good to add it to the readme :) 

Is there any chance you would be willing to share an export of the items you have created ?

**Reply by avorial** [reply]
Mine is very hodge podge.  I think @3darkman does not add it to the world because it is the copyright of Green Ronin. Also I know that a lot may change when he cleans up the weapon bonuses. 
[items.zip](https://github.com/3darkman/foundryvtt-chroniclesystem/files/9610236/items.zip)
This is my items folder from my data.

**Reply by jesuisfrog** [reply]
I'll at least have a base to start with so thank you !
